lang/pl.json

```diff
@@ -67,13 +67,34 @@
     "Open main menu": "Otwórz menu główne",
     "Please verify your credentials and try again": "Sprawdź swoje dane logowania i spróbuj ponownie",
     "Invalid password or username": "Nieprawidłowe hasło lub nazwa użytkownika",
-
+    "Show favorites": "Pokaż ulubione",
+    "City added to favorites!": "Miasto dodane do ulubionych!",
+    "City removed from favorites!": "Miasto usunięte z ulubionych!",
+    "City updated successfully!": "Miasto zaktualizowane pomyślnie!",
+    "Country updated successfully!": "Kraj zaktualizowany pomyślnie!",
+    "City created successfully!": "Miasto utworzone pomyślnie!",
+    "Country created successfully!": "Kraj utworzony pomyślnie!",
+    "City deleted successfully!": "Miasto usunięte pomyślnie!",
+    "Country deleted successfully!": "Kraj usunięty pomyślnie!",
+    "Login to add to favorites!": "Zaloguj się, aby dodać do ulubionych",
+    "There was an error creating city!": "Wystąpił błąd podczas tworzenia miasta!",
+    "There was an error creating country!": "Wystąpił błąd podczas tworzenia kraju!",
+    "There was an error adding alternative city name!": "Wystąpił błąd podczas dodawania alternatywnej nazwy miasta!",
+    "Alternative city name added successfully!": "Alternatywna nazwa miasta dodana pomyślnie!",
+    "Alternative city name deleted successfully!": "Alternatywna nazwa miasta usunięta pomyślnie!",
+    "City providers updated successfully!": "Dostawcy miasta zaktualizowani pomyślnie!",
```

Comment:
"Dostawcy miasta"?  Can we phrase it differently?

Reply:
fixed